src/hooks/useDropdownAutoHeight.ts

```diff
@@ -2,22 +2,21 @@
 
 import { MutableRefObject, useLayoutEffect, useState } from 'react';
 
-export const DEFAULT_DROPDOWN_MAX_HEIGHT = 'auto';
-
 type DropdownAutoHeightProps = {
     isOpen: boolean;
     autoResize?: boolean;
     isDialog?: boolean;
 };
 
 export const getInnerOverlayHeight = (triggerRef: MutableRefObject<HTMLElement | null>) => {
-    let maxHeight = 'auto';
+    const maxHeight: { toTop?: number; toBottom?: number } = {};
```

Comment:
We should think about having a default `auto` value to ensure no breakage?  At least for `toBottom` as this is used as the key returned value in most components.